modules/4337/hardhat.config.ts

```diff
@@ -3,6 +3,7 @@ import 'hardhat-deploy'
 import dotenv from 'dotenv'
 import type { HardhatUserConfig, HttpNetworkUserConfig } from 'hardhat/types'
 import yargs from 'yargs/yargs'
+import { filesToOverride } from './overrides'
```

Comment:
This shouldn't be needed now that we stayed on 0.8.23 right? I would suggest rolling back these changes if that is the case.

Reply:
Done 👍🏾